nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/PutHiveStreaming.java

```diff
@@ -218,6 +218,16 @@
             .addValidator(GREATER_THAN_ONE_VALIDATOR)
             .defaultValue("100")
             .build();
+    
```

Comment:
This line is "empty" but has whitespace in it, causing a CheckStyle violation. I will fix while merging, so no worries.  In the future you can detect such things by adding -Pcontrib-check to your Maven command.